src/test/scala/latis/service/hapi/DataServiceSpec.scala

```diff
@@ -167,4 +192,29 @@ class DataServiceSpec extends FlatSpec {
   it should "reject trailing commas" in {
     csvDecoderReject("a,b,")
   }
+
+  "The DataService" should "correctly generate a response for a CSV request" in {
+    val req = Request[IO](Method.GET, uri"/data?dataset=testdataset&start=2000-01-01&stop=2000-01-06&format=csv")
+    val body = service.orNotFound(req).flatMap { res =>
+      res.bodyText.compile.toList
+    }.unsafeRunSync()
+
+    val bodyStr = body.foldLeft(""){ (acc, s) => acc + s }
+    val testStr = "2000-01-01T00:00:00.000Z,1\r\n2000-01-02T00:00:00.000Z,5\r\n2000-01-03T00:00:00.000Z,4\r\n2000-01-04T00:00:00.000Z,2\r\n2000-01-05T00:00:00.000Z,0\r\n"
```

Comment:
It's not spelled out in the current version of the spec, but based on previous versions I believe the intent is that newlines in the CSV response are just `\n`. Something must be using the platform-independent line separator somewhere.